src/javascript/azriakmalk-CountSquareArea.js

```diff
@@ -0,0 +1,4 @@
+//the function is count square area
+const Square = (n) => {
```

Comment:
@azriakmalk rename the function. Checkout other examples in the src folder

Reply:
@JazibJafri okey